.circleci/config.yml

```diff
@@ -28,8 +28,34 @@ jobs:
       - run:
           name: Run tests
           command: node common/scripts/install-run-rush.js test --verbose
+  update_npm_dependencies:
+    docker:
+      - image: circleci/node:10
+    steps:
+      - checkout
+      - run:
+          name: Install needed npm packages
+          command: npm install npm-check-updates @octokit/rest --no-save
+      - run:
+          name: Configure git
+          command: |
+            git config user.email "auto-npm-dep-update@shiftjs.com"
```

Comment:
Is this email activated, or supposed to be a noreply one?

Reply:
no-reply